server/server.go

```
@@ -3319,17 +3329,21 @@ func (s *StanServer) sendAvailableMessages(cs *stores.ChannelStore, sub *subStat
 	sub.Unlock()
 }
 
-func getNextMsg(cs *stores.ChannelStore, nextSeq, lastSent *uint64) *pb.MsgProto {
+func (s *StanServer) getNextMsg(cs *stores.ChannelStore, subject string, nextSeq, lastSent *uint64) *pb.MsgProto {
 	for {
-		nextMsg := cs.Msgs.Lookup(*nextSeq)
+		nextMsg, err := cs.Msgs.Lookup(*nextSeq)
+		if err != nil {
+			s.log.Errorf("Error looking up message %v:%v (%v)", subject, *nextSeq, err)
+			return nil
+		}
 		if nextMsg != nil {
 			return nextMsg
 		}
 		// Reason why we don't call FirstMsg here is that
 		// FirstMsg could be costly (read from disk, etc)
 		// to realize that the message is of lower sequence.
 		// So check with cheaper FirstSequence() first.
-		firstAvail := cs.Msgs.FirstSequence()
+		firstAvail, _ := cs.Msgs.FirstSequence()
```

Why ignore the error?